signer/tests/integration/transaction_signer.rs

```diff
@@ -742,7 +741,10 @@ mod validate_dkg_verification_message {
 
     impl Default for TestParams {
         fn default() -> Self {
-            let new_aggregate_key = Keypair::new_global(&mut OsRng).x_only_public_key().into();
+            // Here we use explicitly seeded RNG to ensure that [`default()`] is deterministic.
+            // (and default don't allow to pass any parameters)
+            let mut rng = rand::rngs::StdRng::seed_from_u64(51);
+            let new_aggregate_key = Keypair::new_global(&mut rng).x_only_public_key().into();
```

Comment:
It makes sense to have default deterministic, however the tests benefit from randomness. Better to move this implementation away from `Default::default` and into a new `TestParams::new_with_rng` function (or similar) so that it can take a random number generate as an input parameter.